samples/lightdb/get/src/main.c

```diff
@@ -9,14 +9,10 @@ LOG_MODULE_REGISTER(golioth_lightdb, LOG_LEVEL_DBG);
 
 #include <net/golioth/system_client.h>
 #include <samples/common/wifi.h>
-#include <zephyr/net/coap.h>
```

Comment:
Should update the copyright at the top of the file to `2021-2022`.

Reply:
Updated.